lib/link.js

```diff
@@ -10,7 +10,27 @@ const link = (directories, cwd = process.cwd()) => {
 
     fs.exists(dist, (exists) => {
       if (exists) { return; }
-      fs.symlinkSync(src, dist, 'dir');
+
+      fs.stat(cwdSrc, (err, stats) => {
+        if (err) {
+          if (err.code === 'ENOENT') {
+            throw new Error(`No entry: ${cwdSrc}`);
+          }
+          throw err;
+        }
+
+        const type = stats.isFile() ?
```

Comment:
I think in this case it is easy to read, but to extend the number of types in the future, the function would be useful.